activesupport/lib/active_support/cache/file_store.rb

```diff
@@ -26,6 +26,13 @@ def initialize(cache_path, options = nil)
         @cache_path = cache_path.to_s
       end
 
+      # Advertise that this cache store can be used
+      # with "recyclable cache keys" otherwise known
+      # as cache versioning.
+      def supports_in_cache_versioning?
+        true
+      end
+
```

Comment:
Let's do a `supports_cache_versioning?` class method for these.

Comment may be unwrapped to `# Advertise cache versioning support.`

(Worth noting that this duck typing won't cover subclasses of these cache stores.)